asmcomp/x86_proc.ml

```diff
@@ -83,9 +83,12 @@ let string_of_symbol prefix s =
     Buffer.add_string b prefix;
     String.iter
       (function
-        | ('A'..'Z' | 'a'..'z' | '0'..'9' | '_' | '.') as c ->
+        | ('A'..'Z' | 'a'..'z' | '0'..'9' | '_') as c ->
             Buffer.add_char b c
         | c ->
+          if c = Compilenv.symbol_separator () then
```

Comment:
Maybe move this function call outside the `String.iter`

Reply:
Done.